hack/deployer/runner/ocp.go

```diff
@@ -182,38 +182,30 @@ func (d *OcpDriver) Execute() error {
 }
 
 func (d *OcpDriver) auth() error {
+	log.Println("Authenticating as service account...")
 
-	if d.plan.ServiceAccount {
```

Comment:
Removing this line makes accessing vault (and having the service account stored in vault) always required, which means everyone will be using `OcpVaultPath` unless it's configured/overwritten.

Reply:
Yeah I think you have a point here, I'll revert this commit. The switch on `plan.ServiceAccount ` is confusing since using a service account is mandatory so it's a bit confusing. But indeed I can see how it could be useful to use the local gcloud auth if set up correctly.